website/docs/addon_3dequalizer_admin.md

```diff
@@ -12,3 +12,17 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 <ReactMarkdown>
 {versions.Equalizer_Badge}
 </ReactMarkdown>
+
+## Overview
+
+To get 3DEqualizer (3DE) addon running, we need to instal Qt Python bindings - for that we are using pre-launch hook that is automatically installing PySide2/6 into (3DE) Python. You can use ```TDE4_HOME``` environment variable to set the location, or it will be detected from the AYON settings.
+
+:::warning Python Bindings
```

Comment:
Consider the suggestion below.
```suggestion
:::warning Python Bindings not working for 3DE which installed in protected location.
```

Reply:
Thank you. it should be resolved in https://github.com/ynput/ayon-documentation/pull/245/commits/54391d84509d492944b555a2dd5e4501a1d60b72